tests/unit_test/test_unit_agent_inject_user_message.py

```
@@ -0,0 +1,57 @@
+import pytest
```

Flake8 (7.2.0)

[error] 'pytest' imported but unused

(F401)